packages/controller/lib/setup/setupInstall.js

```diff
@@ -25,9 +25,9 @@ function Install(options) {
 
     // todo solve it somehow
     const unsafePermAlways = [
```

Comment:
do we still need it? we have io-package.common.unsafePerm for a long time now and every adapter can set it

Reply:
But do they also do that? Yes we can put as 4.1 topic

Reply:
for xbox I've added, amazon-dash is dead afaik, then just zwave left